resources/lib/addon.py

```diff
@@ -104,8 +104,8 @@ def favorites_refresh():
 
 @plugin.route('/programs')
 @plugin.route('/programs/<program>')
-@plugin.route('/programs/<program>/<season>')
 @plugin.route('/programs/<program>/<season>/')
+@plugin.route('/programs/<program>/<season>')
 def programs(program=None, season=None):
```

Comment:
This breaks urls handled by `ActivateWindow`. I get `No route to path`.

Reply:
I'll remove it from this PR, but maybe we have to fix proper trailing slash handling in the routing library instead :-(

Reply:
And we probably want integration tests for trailing slash handling as well.